testssl.sh

```diff
@@ -355,7 +355,16 @@ HAS_CHACHA20=false
 HAS_AES128_GCM=false
 HAS_AES256_GCM=false
 HAS_ZLIB=false
-OSSL_CIPHERS_S=""
+HAS_DIG=false
+HAS_HOST=false
+HAS_DRILL=false
+HAS_NSLOOKUP=false
+HAS_IDN=false
+HAS_IDN2=false
+HAS_AVAHIRESOLVE=false
+HAS_DIG_NOIDNOUT=false
+
+ OSSL_CIPHERS_S=""
```

Comment:
Style: Extra space in the beginning is probably not supposed to be here, so it's all on one line.  Just being nitpicky at this point, but still.

Reply:
:-)

done